nullaway/src/main/java/com/uber/nullaway/fixserialization/location/AbstractSymbolLocation.java

```diff
@@ -51,9 +53,10 @@ public AbstractSymbolLocation(ElementKind type, Symbol target) {
             + ".");
     this.type = type;
     this.enclosingClass = castToNonNull(ASTHelpers.enclosingClass(target));
-    this.uri =
+    URI pathInURI =
         enclosingClass.sourcefile != null
             ? enclosingClass.sourcefile.toUri()
             : (enclosingClass.classfile != null ? enclosingClass.classfile.toUri() : null);
```

Comment:
Do we still want to get URIs for class files?  Since this version is working, I'm hesitant to make changes here, but maybe we can add a comment to possibly revisit later

Reply:
085be287680dbd317fd64c96a2f48e93141fa461